app/forms/prior_authority/travel_cost_form.rb

```diff
@@ -23,8 +23,10 @@ def save
     private
 
     def process_fields
-      process_field(value: travel_time.to_i, field: 'travel_time')
-      process_field(value: travel_cost_per_hour.to_s, field: 'travel_cost_per_hour')
+      comment_field = 'travel_adjustment_comment'
```

Comment:
This change is _so_ helpful to me for the task I'm doing right now (showing these comments back to the provider) - having travel comments in the payload instead of having to go spelunking through the event log is brilliant!

Reply:
😄 i did try the spelunking. it was a bit of a mare and then i saw that the latest adjustment comment is stored on the payload anyway. The fact that a quote is actually two things combined (service and travel costs) is still a pain which forced me to break the pattern but 🤷